frontend/src/metabase/admin/settings/components/widgets/EmbeddingSwitchWidget/EmbeddingSwitchWidget.tsx

```diff
@@ -0,0 +1,24 @@
+import { t } from "ttag";
+import { Stack, Switch, Text } from "metabase/ui";
+
+interface EmbeddingSwitchWidgetProps {
+  setting: { value: boolean | null };
+  onChange: (value: boolean) => void;
+}
+
+export const EmbeddingSwitchWidget = ({
+  setting,
+  onChange,
+}: EmbeddingSwitchWidgetProps) => (
+  <Stack spacing={"md"} className="text-measure">
+    <Text lh={1.5}>
+      {t`Embed dashboards, questions, or the entire Metabase app into your application. Integrate with your server code to create a secure environment, limited to specific users or organizations.`}
+    </Text>
+    <Switch
+      labelPosition="left"
+      checked={Boolean(setting.value)}
+      label={<strong>{t`Embedding Enabled`}</strong>}
```

Comment:
It seems the copy is now updated to `Enabled Embedding`

Reply:
fixed in https://github.com/metabase/metabase/pull/37043/commits/16a3c3512a4f27e9bdc73d3abeb2e95eeb60c657